WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -449,10 +449,6 @@ - (BOOL)tabBarController:(UITabBarController *)tabBarController shouldSelectView
     if (selectedIndex != tabBarController.selectedIndex) {
         switch (selectedIndex) {
             case WPTabMySites: {
-                if (![Feature enabled:FeatureFlagNewNavBarAppearance]) {
```

Comment:
For context: bypassing the blog list view controller made sense when it was the first VC shown under the "My Site" tab.  With the `newNavBarAppearance` feature flag, the first VC shown under "My Site" is the blog details VC.  The bypassing code thus becomes obsolete as can be deducted since this becomes:

```swift
if false {
    // We only need to bypass the blog list if we're using the old presentation style
    [self bypassBlogListViewControllerIfNecessary];
}
```